projects/demo/src/app/app.component.html

```diff
@@ -81,18 +81,44 @@ <h3>Layers</h3>
     ></fa-layers-text>
   </fa-layers>
 </div>
-<p>Slide and increase the number.</p>
-<input
-  type="range"
-  [min]="1000"
-  [max]="2000"
-  [value]="notificationsCounter"
-  (input)="notificationsCounter = $event.target.value"
-/>
+<mat-grid-list cols="2" rowHeight="20px">
```

Comment:
I'm a bit conflicted about bringing in Angular Material as a dependency. I understand that you try to make the demo app look nicer, but IMO it is better to use simple HTML in the demo app for two reasons:

- Less styles and simpler code in the demo app should make it easier to spot and troubleshoot potential issues as there are less variables at play.
- Smaller maintenance effort moving forward as there are less dependencies to take care of.

We can probably open a new issue to discuss how to improve look and feel of the demo website, but I would prefer to keep this PR focused on the position input and remove code which is not directly related to it. I think here it should suffice to have a borderless table and native HTML select element.

Reply:
I would make it native HTML for now. :) Thank you!